protocol/protocol.proto

```diff
@@ -227,6 +227,12 @@ message SpacePayload {
         Inception inception = 1;
         // channels: sorted by channel_id
         repeated ChannelMetadata channels = 2;
+        SnappedSpaceMedia space_media = 3;
+    }
+
+    message SnappedSpaceMedia {
+        bytes creator_address = 1;
+        EncryptedData data = 2;
```

Comment:
we had a miscommunciation about the rename
the idea is that there is a payload
```
EncryptedData space_image = 3;
```
and then there is a snapshotted version of this payload, where the name matches
`SnappedSpaceImage`


the current pr has a `space_image` getting snapped into a `space_media` object named SpaceMedia, which are not the same



NEW

[12:54](https://hntlabs.slack.com/archives/D03DTUS95UL/p1722455644938149)
the idea is that there is a payload
EncryptedData space_image = 3;
and then there is a snapshotted version of this payload, where the name matches
SnappedSpaceImage
12:54
the current pr has a space_image getting snapped into a space_media object, which are not the same